dist/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" >
```

Comment:
@Emmanuelaaron why do we have this bootstrap link here yet we are to use SASS

Reply:
This was just an oversight. let me remove it.

Reply:
@HabibSentongo , I have realized that that link was created automatically only that I needed to include the dist folder inside the .gitignore file

Reply:
Alright great, other things are ok according to me